dashi/rnaseqc/over_time.py

```diff
@@ -132,44 +315,102 @@ def create_subplot(rna_df, graph_list):
                 dcc.DatePickerRange(
                     id='date_picker',
-                    min_date_allowed=min(rna_df['Run Date']),
-                    max_date_allowed=max(rna_df['Run Date']),
-                    start_date=min(rna_df['Run Date']),
-                    end_date=max(rna_df['Run Date']),
+                    display_format='YYYY-MM-DD',
+                    min_date_allowed=min(RNA_DF['Run Date']),
+                    max_date_allowed=max(RNA_DF['Run Date']),
+                    start_date=min(RNA_DF['Run Date']),
+                    end_date=max(RNA_DF['Run Date']),
                 ),
                 html.Br(),
+                html.Br(),
                 html.Label('Show Graphs:'),
                 dcc.Dropdown(
                     id='graphs_to_plot',
                     multi=True,
-                    options=[{'label': x, 'value': x} for x in graphs_to_plot],
-                    value=graphs_to_plot[:4]
+                    options=[
+                        {'label': x, 'value': x} for x in METRICS_TO_GRAPH
+                    ],
+                    value=METRICS_TO_GRAPH[:4]
                 ),
+                html.Br(),
+                html.Div([
+                    html.Div([
+                        html.Label('Colour by:'),
+                        dcc.Dropdown(
+                            id='colour_by',
+                            options=[
+                                {'label': x['name'], 'value': x['id']} for x
+                                in SHAPE_COLOUR_COLUMN
+                            ],
+                            value='Study Title'
+                        ),
+                    ], style={'width': '45%', 'display': 'inline-block', }),
+                    html.Div([
+                        html.Label('Shape by:'),
+                        dcc.Dropdown(
+                            id='shape_by',
+                            options=[
+                                {'label': x['name'], 'value': x['id']} for x
+                                in SHAPE_COLOUR_COLUMN
+                            ],
+                            value='geo_prep_kit'
+                        ),
+                    ], style={
+                        'width': '45%', 'display': 'inline-block',
+                        'float': 'right'
+                    }),
+                ])
             ], style={'margin': '23px'})]
         ),
-        sd_material_ui.RaisedButton(id='filter_button', label='Filters'),
+        html.Div([
+            html.Div(
+                sd_material_ui.RaisedButton(
+                    id='filter_button', label='Options'
+                ), style={'display': 'inline-block', }
+            ),
+            html.Div(
+                sd_material_ui.RaisedButton(
+                    id='update_from_table_button', label='Update Order',
+                ), style={'display': 'inline-block', 'margin-left': '15px'}
+            ),
+        ]),
```

Comment:
These RaisedButtons overlap slightly with the chart top, so they end up looking like tabs (and it's not immediately obvious that they're clickable). Can the chart be moved to eliminate the overlap?

Reply:
Added a small margin at the bottom in bce269f